src/main.rs

```diff
@@ -45,6 +54,8 @@ fn main() {
     // See https://github.com/clap-rs/clap/blob/v2.33.1/examples/12_typed_values.rs
     let max_colours = value_t!(matches, "MAX-COLOURS", usize).unwrap_or_else(|e| e.exit());
 
+    let colour_output = value_t!(matches, "output", ColourOutput).unwrap_or_else(|e| e.exit());
```

Comment:
Maybe move this up a line to pair it with `max_colours`, and tweak the comment? They both use the same pattern.